libs/blocks/icon-block/icon-block.css

```diff
@@ -346,6 +346,10 @@
   gap: 16px;
 }
 
+html[dir="rtl"] .icon-block.inline .foreground .text-content {
+  text-align: right;
+}
+
```

Comment:
I think a better approach to all of this is [to correct the usage of `text-align: left`](https://github.com/adobecom/milo/blob/7c206180637964553aaa4e59ba6701b93a694fbf/libs/blocks/icon-block/icon-block.css#L345) to be `text-align: start`

That way when the `dir` attribute is applied the direction is automatically adjusted based on `dir`.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

Reply:
@rgclayton I have updated the code.